app/views/unattended/provisioning_templates/snippet/preseed_kernel_options.erb

```diff
@@ -32,7 +38,12 @@ description: options for the kernel / preseed startup initialization
   if subnet4 && subnet4.dhcp_boot_mode?
     options << 'netcfg/disable_dhcp=false'
   elsif subnet4 && !subnet4.dhcp_boot_mode?
-    options << 'netcfg/disable_dhcp=true netcfg/get_ipaddress=${netX/ip} netcfg/get_netmask=${netX/netmask} netcfg/get_gateway=${netX/gateway} netcfg/get_nameservers=${dns} netcfg/confirm_static=true'
+    if @ipxe_net
```

Comment:
where is this (`@ipxe_net`)) defined?

The only definition I see is in `app/views/unattended/provisioning_templates/iPXE/kickstart_default_ipxe.erb`, but that does not load the *preseed* snippet.

Reply:
You are right, we tried to get it more consistent with the kickstart-templates but the variable is missing in the preseed iPXE-template

So I need to add that variable in the iPXE-template as well to define it - I will check if we have that already or if I need to retest something

Reply:
For kickstart, it is explicitly set when using iPXE:

```
app/views/unattended/provisioning_templates/iPXE/kickstart_default_ipxe.erb:kernel <%= "#{@host.url_for_boot(:kernel)}" %> initrd=initrd.img <%= pxe_kernel_options %> <%= snippet("kickstart_kernel_options", variables: {ipxe_net: true}).strip %>
```

Other invocations of the same snippet use `<%= snippet("kickstart_kernel_options").strip %>` without passing extra vars.

I guess you'd need a smilar change to `app/views/unattended/provisioning_templates/iPXE/preseed_default_ipxe.erb`?

Reply:
Exactly, but I am not sure if we already implemented it in our tests after the changes and I just forgot to add it the PR or if I need to add it and I should retest if all those provisioning variants still work after that test (which should not need much time but I need to find the time for it)

Reply:
I updated the iPXE-template and changed the preseed_kernel_options that it still has the BOOTIF-option like before, so that there are not two bootifs and the configuration is still as it was before